wandb/proto/wandb_internal.proto

```diff
@@ -44,7 +44,6 @@ message Record {
     HeaderRecord header = 21;
     FooterRecord footer = 22;
     RunPreemptingRecord preempting = 23;
-    LinkArtifactRecord link_artifact = 24;
```

Comment:
Per proto best practices, you'd need to mark this field number `reserved`. BUT see my other comment---you can't delete this record. If `wandb sync` behavior doesn't matter, you can replace the type by `google.protobuf.Empty` and keep the field here with a comment. It is also safe to rename it.

```proto
import "google/protobuf/empty.proto";

message Record {
  oneof record_type {
    // Removed but kept to avoid panics in `wandb sync`.
    google.protobuf.Empty noop_link_artifact = 24;
  }
}
```

Reply:
Interesting, i wasn't aware of this rule. Thanks for the info

Reply:
Do I need to do anything different around this in the handler? Or will it just be ignored because of `google.protobuf.Empty`?

Reply:
@timoffex i added this import to the internal proto file: `import "google/protobuf/empty.proto";` and its causing this CI error: 
```
pkg/service/wandb_internal.pb.go:12:2: cannot find module providing package google.golang.org/protobuf/types/known/emptypb: import lookup disabled by -mod=vendor
Traceback (most recent call last):
  File "<string>", line 11, in <module>
  File "/root/.cache/uv/environments-v0/.tmp3EkoBY/lib/python3.10/site-packages/hatchling/build.py", line 58, in build_wheel
    return os.path.basename(next(builder.build(directory=wheel_directory, versions=['standard'])))
  File "/root/.cache/uv/environments-v0/.tmp3EkoBY/lib/python3.10/site-packages/hatchling/builders/plugin/interface.py", line 147, in build
    build_hook.initialize(version, build_data)
  File "/root/project/hatch_build.py", line 32, in initialize
    self._prepare_wheel(build_data)
  File "/root/project/hatch_build.py", line 38, in _prepare_wheel
    artifacts.extend(self._build_wandb_core())
  File "/root/project/hatch_build.py", line 123, in _build_wandb_core
    hatch_core.build_wandb_core(
  File "/root/project/core/hatch.py", line 39, in build_wandb_core
    subprocess.check_call(
  File "/usr/local/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/root/go/bin/go', 'build', '-ldflags=-s -w -X main.commit=f644f04386c6708ebf9ef00dc93f5af295da9761 -extldflags "-fuse-ld=gold -Wl,--weak-unresolved-symbols"', '-o', '../wandb/bin/wandb-core', '-mod=vendor', 'cmd/wandb-core/main.go']' returned non-zero exit status 1.
---
```

I was able to fix it locally but how can i fix it to avoid CI issues?

Reply:
Ah, you need to run `go mod vendor` in the `core/` directory. It will pull the `emptypb` package and add it to the `core/vendor/` directory.

Reply:
Ah thanks - I think that fixed it!